dist/commonjs/router/router.js

```diff
@@ -55,7 +63,7 @@ function getTransitionByIntent(intent, isIntermediate) {
     }
 
     // No-op. No need to create a new transition.
-    return new Transition(this);
+    return this.activeTransition || new Transition(this);
```

Comment:
@jmeas sounds reasonable, that way the diffs are cleaner when we have to do a bisect.
Unrelated to this PR, we should likely explore not committing built assets to the repo.